app/services/licences/fetch-licence.service.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+/**
+ * Fetches data needed for the bill run page which includes a summary for each bill linked to the bill run
```

Comment:
Damn you copy & paste!

```suggestion
 * Fetches data needed for the view '/licences/{id}` page
```